### DISCUSSION TITLE
Inverse problem of a purely ellipsoidal light curve

### DISCUSSION THREAD

**Discussion by roygomel** [phoebe-project/phoebe2#604] [Questions]
Dear Phoebe team,

I have an observational data of a light curve that presents the well-known ellipsoidal modulation, without any eclipses. We can also assume a circular orbit for simplicity.

Is it possible to solve the inverse problem for this kind of light curve, and if so, could you please direct me how to implement a code?

Thanks a lot,
Roy

**Comment by aprsa** [comment]
It is certainly possible to solve the light curve but your solution will in most circumstances be extremely degenerate. Ellipsoidal variations arise from tidal and rotational surface distortion *and* surface brightness distribution (gravity, limb darkening) *and* it is an integrated effect, so many combinations of parameters will result in an indistinguishable light curve model.

Your ability to solve such a system thus depends on auxiliary data. If you have RVs, spectra, distances, interferometry, etc, these may break the degeneracy to *some* extent, where "some" depends on the data and the modeled system.

If you tell us a little bit more about your particular system, we may be able to provide more specific thoughts.

**Comment by roygomel** [comment]
Dear Prsa,

In a previous paper (Gomel, Mazeh, and Faigler 2021) we introduced a new quantity named "modified Minimum Mass Ratio" (mMMR). This quantity is always smaller than the actual mass ratio of the system (defined as secondary-to-primary mass ratio) and thus binaries with mMMR > 1 might be good candidates to harbor a compact secondary if the primary is a main-sequence star. Otherwise, we would expect to see the over-luminous regular-star companion.

The method is quite robust since the mMMR is directly derivable from the ellipsoidal semi-amplitude of a light curve, and does not depend on the primary radius and mass (that in many cases cannot be accurately estimated).

As part of Gaia third data release (DR3) we have made a search for compact-companion candidates by looking at G-band light curves that present large ellipsoidal variations. We derived the mMMR for each system and focused on cases with mMMR larger than unity. This work is planned to be published on June 14, and till then we are not allowed to present specific details on these systems.

For these systems, we know the binary's orbital period and the Gaia G-band measured light curve. We could also use estimates for the primary teff, mass, and radius from Gaia. Now, we wish to use your fantastic PHOEBE code to best fit the observed light curves of a few candidates. For simplicity, we can start with a model of a normal star and a compact secondary (we could simply set its radius close to zero), assume a circular orbit (as our systems have an orbital period smaller than 2.5 days), and use the orbital period of Gaia and, if needed, add constraints on the primary teff, mass, and radius.

We would be grateful to have your kind help in implementing such code.

Thanks a lot,

Roy Gomel

**Reply by kecnry** [reply]
Hi Roy - 

I'll just add that phoebe does [support components with mass but that don't contribute any light or eclipsing effects](http://phoebe-project.org/docs/latest/examples/distortion_method_none) that might accomplish what you want to do (setting a radius close to 0 would attempt to still compute the intensities and the logg values would likely result in a lot of atmosphere out-of-bounds errors).

-Kyle

**Comment by roygomel** [comment]
Yes, indeed that would be very helpful. 
Please let me know if you could help me produce such an ellipsoidal light curve fitting code (see my previous comment).
Best,
Roy

**Reply by kecnry** [reply]
I'm happy to answer questions if you run into any issues, but unfortunately don't have the time to write up a full example for your exact use-case. I think all the pieces you need should be covered in the existing tutorials though.
